uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp-leave-absences.js

```diff
@@ -0,0 +1,64 @@
+var Promise = require('es6-promise').Promise;
+var page = require('./page');
+
+module.exports = (function () {
+
+  return page.extend({
+
+    /**
+     * [openMyReport description]
+     * @return {[Page]} [description]
+     */
+    openMyReport: function () {
```

Comment:
`openMyReport` is same as waiting for the page to load. Please add this to `waitForReady` function which automatically gets fired when init is called.